src/timer.js

```diff
@@ -1,10 +1,11 @@
 import { formatError } from "./utils.js";
-import "./howler.js"
+import "../node_modules/howler/src/howler.core.js";
```

Comment:
Еще одна прелесть вебпака это то что, он нам позволяет импортировать из commonJs модулей как будто они es-модули. Поэтому теперь можно импортировать из Howler как из Luxon)  
```suggestion
import { Howl } from "howler";
```